src/msg/async/AsyncConnection.cc

```diff
@@ -257,9 +257,10 @@ ssize_t AsyncConnection::read_until(unsigned len, char *p)
   }
 
   ssize_t r = 0;
+  uint64_t to_read = 0;
```

Comment:
if to_read is not passing a value between these 3 sites, better to use 3 different locals